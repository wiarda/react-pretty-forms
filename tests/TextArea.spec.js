/* global describe test expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { TextArea } from '../index';

describe('TextArea component tests', () => {
  test('Renders consistently', () => {
    const component = renderer.create(<TextArea name="Long" required label="Essay" />).toJSON();
    expect.assertions(1);
    expect(component).toMatchSnapshot();
  });
});
