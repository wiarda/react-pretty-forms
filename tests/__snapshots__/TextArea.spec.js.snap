// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea component tests Renders consistently 1`] = `
<div
  className="pretty-row"
  style={Object {}}
>
  <label
    className="pretty-label-textarea"
    data-active={false}
    data-validity={true}
    htmlFor="Long"
  >
    <span
      className="pretty-label-text-textarea"
      data-active={false}
    >
      Essay
    </span>
  </label>
  <textarea
    className="pretty-textarea"
    data-validity={true}
    name="Long"
    onBlur={[Function]}
    onFocus={[Function]}
    required={true}
  />
  <div
    className="pretty-validation"
    data-validity={true}
  >
    Please fill in this required field
  </div>
</div>
`;
